shared/src/main/scala/doodle/core/BoundingBox.scala

```diff
@@ -64,16 +62,37 @@ object BoundingBox {
       val xAmount = math.abs(x) / 2
       val yAmount = math.abs(y) / 2
       BoundingBox(
-        inner.left   - xAmount,
-        inner.top    - yAmount,
-        inner.right  + xAmount,
-        inner.bottom + yAmount
-      )
+        inner.left - xAmount,
+        inner.top - yAmount,
+        inner.right + xAmount,
+        inner.bottom + yAmount)
 
-    case ContextTransform(f, i) =>
+    case ContextTransform(f, i) ⇒
       BoundingBox(i)
 
-    case d: Drawable =>
+    case d: Drawable ⇒
       BoundingBox(d.draw)
+
+    case Path(head, tail) ⇒
```

Comment:
I'm getting:

```
[warn] doodle/js/src/main/scala/doodle/js/package.scala:50: match may not be exhaustive.
[warn] It would fail on the following input: Path(_, _)
[warn]     img match {
[warn]     ^
```

...so presumably there's something missing from this implementation over in that file?